components/automate-deployment/pkg/target/local_target.go

```diff
@@ -81,6 +81,7 @@ Restart = on-failure
 LimitNOFILE = 65536
 LimitMEMLOCK=infinity
 UMask = 0022
+TasksMax = infinity
```

Comment:
Presumably this limit exists for a reason. Are we sure we want to set it to unlimited instead of a considered limit?

Reply:
This is a good question.  According to https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html:

> Defaults to 15%, which equals 4915 with the kernel's defaults on the host, but might be smaller in OS containers.

Given that we haven't seen this problem for non SLES users, we could set it to this, or some "round" number like `8192` to give us plenty of headroom.

One "unknown" bit here is that the new Builder component uses MANY threads in its default configuration.